src/commands/misc/fox.js

```diff
@@ -0,0 +1,18 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+const snekfetch = require('snekfetch')
+
+module.exports = class Fox extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'fox'
```

Comment:
missing this.category
```suggestion
    this.name = 'fox'
    this.category = 'misc'
```